packages/react/src/components/ContentBlockMedia/__stories__/ContentBlockMedia.stories.js

```diff
@@ -158,26 +158,22 @@ WithAsideElements.story = {
 
         const linkListProps = {
           heading: text('link list heading:', 'Tutorials', groupId),
```

Comment:
Hey @andysherman2121, I know you didn't touch here for this cleanup, but just a suggestion.... if you change the `link list heading ` to `Link List (heading)`, what do you think?

Reply:
Good idea @Diego-Marcuz. I'll add that now